jmix-core/core/src/main/java/io/jmix/core/CorsProperties.java

```diff
@@ -55,6 +55,18 @@ public class CorsProperties {
      */
     List<String> allowedMethods;
 
+    /**
+     * <p>A list of response headers other than simple headers (i.e. Cache-Control, Content-Language, Content-Type,
```

Comment:
This javadoc comment will go to Spring boot [configuration metadata](https://docs.spring.io/spring-boot/docs/current/reference/html/configuration-metadata.html) and will be displayed in IntelliJ Idea Ultimate as a hint, when you start entering the application property name in your applicaition. Please remove the leading <p> tag to make this hint look good.

Reply:
As far as I understand, you meant this

<img width="535" alt="Screenshot 2023-12-03 at 13 57 31" src="https://github.com/jmix-framework/jmix/assets/51899225/69b96b38-b62a-4014-9a30-9dcc6a1d230d">

Updated javadoc. Now:

<img width="548" alt="Screenshot 2023-12-03 at 14 00 25" src="https://github.com/jmix-framework/jmix/assets/51899225/8b5f8b69-d4c0-4186-88fc-ad2844b86542">

<img width="738" alt="Screenshot 2023-12-03 at 13 58 54" src="https://github.com/jmix-framework/jmix/assets/51899225/6232427f-0e2a-4a95-ac05-9f0f85ef43f4">

Reply:
Exactly) Thank you